app/poros/food.rb

```diff
@@ -0,0 +1,15 @@
+class Food
+  attr_reader :total_hits,
```

Comment:
Adding total_hits here is a clever solution to grabbing that data. Keep in mind that you can also return more than one object from a ruby method, so in your facade, you could return `[foods, food_data[:totalHits]` and then use `.first` and `.last` in you controller to distinguish between those 2 objects